tools/java/org.hl7.fhir.tools.core/src/org/hl7/fhir/tools/publisher/Publisher.java

```diff
@@ -336,7 +337,7 @@ public String getPath() {
   private String singlePage;
 
   private Map<String, Example> processingList = new HashMap<String, Example>();
-  

```

Comment:
There are a *lot* of white-space changes in this file.  While I agree that your version is better, it makes it difficult to see what really changed.  Perhaps it's too late for this, but if you're able to back this change out and only change the lines related to your ember generator, that might be best-- especially if we ever try to get this accepted into the real FHIR source code repo.